src/components/SearchBar/index.jsx

```diff
@@ -0,0 +1,93 @@
+import { Component } from 'react';
+import { func, number } from 'prop-types';
+import { searchBar, searchInput, filterLinesIcon } from './index.module.css';
+import filterLinesSvg from './icons/filter-lines.svg';
+
+export default class SearchBar extends Component {
+  static propTypes = {
+    /**
+     * Executes a function when the user has typed search keywords
+     * and presses enter.
```

Comment:
Is this comment correct? Should the user press enter explicitly to execute the search?

Reply:
That was the first approach :) Going to change it.